src/Sylius/Bundle/CoreBundle/Import/Writer/ORM/UserWriter.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Import\Writer\ORM;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Psr\Log\LoggerInterface;
+use Sylius\Bundle\CoreBundle\Import\Writer\ORM\Processor\UserProcessorInterface;
+use Sylius\Component\Core\Model\UserInterface;
+use Sylius\Component\ImportExport\Model\JobInterface;
+use Sylius\Component\ImportExport\Writer\WriterInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class UserWriter implements WriterInterface
+{
+    /**
+     * @var ObjectManager
+     */
+    private $entityManager;
+    /**
+     * @var UserRepositoryInterface
+     */
+    private $userRepository;
+    /**
+     * @var RepositoryInterface
+     */
+    private $customerRepository;
+    /**
+     * @var int
+     */
+    private $resultCode = 0;
+    /**
+     * @var UserProcessorInterface
+     */
+    private $userProcessor;
+
+    /**
+     * @param UserProcessorInterface  $userProcessor
+     * @param UserRepositoryInterface $userRepository,
+     * @param RepositoryInterface     $customerRepository,
+     * @param ObjectManager           $entityManager
+     */
+    public function __construct(
+        UserProcessorInterface $userProcessor,
+        UserRepositoryInterface $userRepository,
+        RepositoryInterface $customerRepository,
+        ObjectManager $entityManager
+    ) {
+        $this->userProcessor = $userProcessor;
+        $this->userRepository = $userRepository;
+        $this->customerRepository = $customerRepository;
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getResultCode()
+    {
+        return $this->resultCode;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
+    {
+        return 'user_orm';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function write(array $rawUsers, array $configuration, LoggerInterface $logger)
```

Comment:
It should be used Another thing is, that interface force to have it. Just to be sure, that developer will do something with it :D